index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
```

Comment:
- [ ] Good job importing the fonts!  However, you didn't import any font-weights, and that's why the font-weight 700 is not displaying on the page.

Go to the [Poppings font page](https://fonts.google.com/specimen/Poppins) and select all the font weights that your project needs (400, 500, 700 are the most used, but you could add more if the project requires them).  Then copy the selected 3 links to the`<head>` section of your HTML file:

![image](https://user-images.githubusercontent.com/57421823/149146506-79b61113-22f2-439d-91c3-381c4753a190.png)

Then:

![image](https://user-images.githubusercontent.com/57421823/149147299-29ea015d-639d-4a53-94e0-8eeef69d237c.png)

Now please look at the last `<link>`:

```HTML
<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet"> 
```
And notice the part that says 'wght@400;500;700'.  That way, you can import all the font weights you like, and they will be actually displayed on the page!! 💪  I hope this makes sense to you 👍